lib/v2/gamegene/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'gamegene.cn/': {
+        _name: '游戏基因 GameGene',
```

Comment:
```suggestion
        _name: '游戏基因',
```
`_name` should follow the heading level in your docs.